packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/TreeSitterScopeHandler.ts

```diff
@@ -117,29 +117,73 @@ export class TreeSitterScopeHandler extends BaseScopeHandler {
   }
 
   private matchToScope(editor: TextEditor, match: QueryMatch): TargetScope {
+    const scopeTypeType = this.scopeType.type;
+
     const contentRange = getNodeRange(
-      match.captures.find((capture) => capture.name === this.scopeType.type)!
-        .node,
+      match.captures.find((capture) => capture.name === scopeTypeType)!.node,
+    );
+
+    const domain =
+      getRelatedRange(match, scopeTypeType, "domain") ?? contentRange;
+
+    const removalRange = getRelatedRange(match, scopeTypeType, "removal");
+
+    const leadingDelimiterRange = getRelatedRange(
+      match,
+      scopeTypeType,
+      "leading",
+    );
+
+    const trailingDelimiterRange = getRelatedRange(
+      match,
+      scopeTypeType,
+      "trailing",
     );
 
+    const interiorRange = getRelatedRange(match, scopeTypeType, "interior");
```

Comment:
Should we call this `"inside"`?  That's the term everyone knows.  I'd also be tempted to do a rename `interior` => `inside` in our code base, so maybe we should just do that in a follow-up